app/(company)/run-for-office/components/Hero.js

```diff
@@ -1,39 +1,56 @@
 'use client';
-import Pill from '@shared/buttons/Pill';
 import MaxWidth from '@shared/layouts/MaxWidth';
-import Chat from './Chat';
 import RunCampaignButton from './RunCampaignButton';
+import SecondaryButton from '@shared/buttons/SecondaryButton';
+import HeroImages from './HeroImages';
+import Image from 'next/image';
 
 export default function Hero({ demoCallback }) {
   return (
     <MaxWidth>
-      <div className="grid grid-cols-12 lg:gap-10 py-16">
-        <div className="col-span-12 lg:col-span-6">
-          <h1 className="font-black text-6xl">
-            Free tools and expertise to run winning campaigns
+      <div className="flex flex-row w-full">
+        <div className="flex flex-row w-full mt-5 justify-between h-auto">
+          <Image
+            src="/images/run-for-office/squiggles.svg"
+            width="90"
+            height="90"
+            className="ml-12"
+          />
+
+          <Image
+            src="/images/run-for-office/hero-star.svg"
+            width="120"
+            height="120"
+            className="mt-5"
+          />
+        </div>
+      </div>
+
+      <div className="grid grid-cols-12 md:gap-10 pl-8 pr-8">
+        <div className="col-span-12 md:col-span-6">
```

Comment:
This is the way
https://i.kym-cdn.com/photos/images/newsfeed/001/653/546/4b2.jpg